docs/Getting_Involved/Review.rst

```diff
@@ -1,13 +1,13 @@
 Reviewing
 =========
 
-This document is a guide to coalas review process. It will be extended
+This document is a guide to *coala's* review process. It will be extended
```

Comment:
i have thought about this a little too, but i think it would look ugly, let me test it